corehq/motech/repeaters/tests/test_models.py

```diff
@@ -1,17 +1,23 @@
+from django.conf import settings
 from django.test import TestCase
+from nose.tools import assert_in
 
 from corehq.motech.const import BASIC_AUTH
 from corehq.motech.models import ConnectionSettings
-from corehq.motech.repeaters.models import (
```

Comment:
What's the rationale for these relative imports? I don't know if that's a pattern we generally use and I find it a bit harder to parse.

Reply:
I think it was @millerdev who I saw using relative imports. At first I didn't like them either. ...

... but I adopted them because **isort** separates out imports from the same Django app from imports from other apps, so it's clearer what belongs with what. -- You'll see that I only do it for modules in the same Django app, so `corehq.motech.repeaters.models` is a relative import, but `corehq.motech.models` is not.

My hope was to make imports easier to parse. If you don't think that's working, I'm willing to revert, but I'd prefer to do that in a later PR, to avoid a bunch of conflicts.

Reply:
I rarely read imports so I don't think it's necessarily worth changing, but I was curious for the rationale. Thanks!